Gems/SimulationInterfaces/Code/Source/Clients/SimulationEntitiesManager.cpp

```diff
@@ -600,6 +616,24 @@ namespace SimulationInterfaces
             }
             const AZ::Entity* root = *view.begin();
 
+            for (auto* entity : view)
+            {
+                ROS2::ROS2FrameComponent* frameComponent = entity->template FindComponent<ROS2::ROS2FrameComponent>();
+                if (frameComponent)
+                {
+                    const AZStd::string f = frameComponent->GetFrameID();
+                    if (f.empty())
+                    {
+                            frameComponent->SetFrameID(name);
+                    }
+                    else
+                    {
+                            frameComponent->SetFrameID(AZStd::string::format("%s/%s", name.c_str(), f.c_str()));
```

Comment:
Not sure if it was your intention, but shouldn't it be like this to apply namespace?
```suggestion
                            frameComponent->SetFrameID(AZStd::string::format("%s/%s", entityNamespace.c_str(), f.c_str()));
```